prog/weavewait/main.go

```diff
@@ -1,32 +1,54 @@
 package main
 
 import (
+	"bytes"
 	"fmt"
+	"io"
+	"net"
 	"os"
 	"os/exec"
 	"os/signal"
+	"strings"
 	"syscall"
 
-	"github.com/weaveworks/weave/net"
+	weavenet "github.com/weaveworks/weave/net"
 )
 
 func main() {
 	if len(os.Args) <= 1 {
 		os.Exit(0)
 	}
-	args := os.Args[1:]
+
+	var (
+		args         = os.Args[1:]
+		notInExec    = true
+		rewriteHosts = true
+	)
 
 	if args[0] == "-s" {
```

Comment:
Seems like this is getting to the stage where it might need 'proper' argument parsing, usage etc.